internal/pkg/cli/svc_deploy.go

```diff
@@ -336,6 +340,16 @@ func workloadManifest(in *workloadManifestInput) (interface{}, error) {
 	if err != nil {
 		return nil, fmt.Errorf("apply environment %s override: %s", in.envName, err)
 	}
+	serviceInRegion, err := mft.IsServiceAvailableInRegion(in.targetEnv.Region)
+	if err != nil {
+		return nil, fmt.Errorf("%s available for region %s: %w ", in.svcType, in.targetEnv.Region, err)
+	}
+
+	if !serviceInRegion {
+		log.Warningf(`%s might not be available in region %s, proceed with caution
```

Comment:
another tiny nit (for here and elsewhere):
```suggestion
		log.Warningf(`%s might not be available in region %s; proceed with caution
```